docs/src/main/sphinx/connector/hive.rst

```diff
@@ -559,7 +559,7 @@ Property Name                                        Description
                                                      where Trino is running, defaults to ``false``.
```

Comment:
commit title too long. Move `Reduces awshttpsclientpoolpendingcount and improves query speed.` to new line

`Reduces awshttpsclientpoolpendingcount and improves query speed.`

1. Under what circumstances?
2. Reduces what?

Reply:
Amended commit message. Thanks.